src/styles/index.scss

```diff
@@ -0,0 +1,45 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&family=Roboto+Slab:wght@300;400;500;600&display=swap');
+@import './navbar.scss';
+@import './book.scss';
+@import './input.scss';
+
+:root {
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --black-5: rgba(0, 0, 0, 0.05);
+  --z: #fafafa;
+  --black-two: #121212;
+  --white: #e8e8e8;
+  --white-two: #fff;
+  --medium-blue: #307bbe;
+  --cool-blue: #4386bf;
+  --azure: #0290ff;
+}
+
+.Bookstore-CMS {
+  width: 100rem;
+  height: auto;
+  padding: 3.313rem 3.75rem 7.688rem 3.75rem;
+  background-color: var(--pale-grey);
+
+  .panel-bg {
+    width: 87.5rem;
+    height: auto;
+    margin: 0 2.5rem 2.063rem 0;
+    padding: 0 0 4.625rem;
+    border-radius: 4px;
+    box-shadow: 0 0 90px 0 var(--black-5);
+    background-color: var(--z);
+
+    .horizontal-divider {
+      width: 76rem;
+      height: 0.115rem;
+      border: solid 2px var(--white);
+      margin: 3.75rem 5.795rem 0.125rem 5.795rem;
+      background-color: #c4c4c4;
+    }
+  }
+}
```

Comment:
- [ ] Please as you can see from the video below, Your project is having horizontal overflow which makes your `Add Book` button and `user-icon` not show on the page unless the page is scrolled horizontally. Please make sure you remove the horizontal overflow and make the design looks exactly like the Zeplin template :relaxed:

[Screencast from 03-04-2023 03:27:03 AM.webm](https://user-images.githubusercontent.com/92303891/222873568-5fd4dc60-b7e4-410a-a3ec-e85cb14ad6dd.webm)

Reply:
Thank you Mr. @Owusu-Desmond ! I'm working on it!